### PR TITLE
Features js

### PR DESCRIPTION
**Pull request by Alzubair98**
add the popup card to the mobile version and desktop version

please ignore the layout of mobile version 
the reason is (week one = low experience);

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+`;
+
+// popupcard function
+
+// for (let j = 0; j < list.length; j++){
+//   list[j].addEventListener("click", function () {
+//     popCard.classList.toggle("active");
+//   })
+// }
+
```
Comment (zieeco) [inline js/index.js:295]: **[REQUIRED CHANGES]**

- Please, kindly remove this commented unused code block, they make your code base not clean and professional.

File: style.css [hunk 2]
```diff
@@ -686,14 +686,179 @@ ul {
...
+/* full card  */
+
+.full-card-cont {
+  width: 80%;
+  margin-left: 10%;
+  backdrop-filter: blur(15px);
+
+  /* display: none; */
+}
+
```
Comment (zieeco) [inline style.css:698]: **[REQUIRED CHANGES]**

- Please, kindly ensure that the pop-up for mobile shows the see live and see source button.

*your design*
hidden button

*figma mobile popup*
mob

Comment (zieeco) [reply style.css:698]: **[REQUIRED CHANGES]**

- Please, kindly ensure your desktop pop header and technologies are well-styled to match the Figma design below:

*figma*
figmaheader

*yours*
yourheader

Comment (zieeco) [reply style.css:698]: **[REQUIRED CHANGES]**

**README FILE**

- Please, kindly update your readme to have the current project name.
- Add a getting started header, which tells the users how to have your project locally on their machine.
- Add a second author that has contributed to this project.
- Remove the headers that are not used to make your readme more professional.
- Update the live demo link as what you have is not showing your current project.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (Alzubair98) [reply js/index.js:295]: done

File: style.css (hunk 2 above)
Comment (Alzubair98) [reply style.css:698]: done

**Pull request merged**